devel/fortio/cmd/echosrv/echo.go

```diff
@@ -23,11 +23,10 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"net/http"
 	"sync/atomic"
 
-	"github.com/golang/glog"
+	log "istio.io/istio/devel/fortio"
```

Comment:
log.DbgOn() was shorter and clearer that it's about logging (I guess this stems from the fact that in theory it should be it's own package - or better, not reinvented - but fortio is sort of a bundle of features that can be used as a whole or piecemeal)